src/control/server/storage/bdev/backend.go

```diff
@@ -207,6 +229,10 @@ func (b *spdkBackend) Format(pciAddr string) (*storage.NvmeController, error) {
 
 	ctrlr, err := getController(pciAddr, b.binding.controllers)
 	if err != nil {
+		if err == FaultPCIAddrNotFound(pciAddr) {
+			return new(storage.NvmeController), nil
+		}
```

Comment:
Just curious what the end result of this is. Seems odd to return no error if the requested PCI address doesn't exist.

Reply:
this is an interim hack until I implement format of backing ssds